docs/ui/workspaces.md

```diff
@@ -0,0 +1,104 @@
+
+## Workspace settings
+
+Within a workspace, select **Workspace Settings** to view or edit workspace details.  
+
+![Managing a workspace in the Prefect Cloud UI.](/img/ui/workspace-settings.png)
+
+This page enables you to copy the configuration string to [set up a local development or execution environment](/ui/cloud-getting-started/#configure-execution-environment) to work this workspace.
+
+The options menu enables you to edit workspace details or delete the workspace.
+
+!!! warning "Deleting a workspace"
+    Deleting a workspace deletes all deployments, flow run history, work queues, and notifications configured in workspace.
+
+## Workspace collaborators
+
+Personal account users may invite _workspace collaborators_, users who can join, view, and run flows in your workspace.
+
+In your workspace, select **Workspace Collaborators**. If you've previously invited collaborators, you'll see them listed.
+
+![Managing collaborators in a workspace in the Prefect Cloud UI.](/img/ui/workspace-collaborators.png)
+
+To invite a user to become a workspace collaborator, select the **+** icon. You'll be prompted for the email address of the person you'd like to invite. Add the email address, then select **Send** to initiate the invitation. 
+
+If the user does not already have a Prefect Cloud account, they will be able to create one when accepting the workspace collaborator invitation.
+
+To delete a workspace collaborator, select **Remove** from the menu on the left side of the user's information on this page.
+
+## Workspace sharing <span class="badge orgs"></span>
+
+Within a Prefect Cloud [organization](/ui/organizations/) organization, Admins and workspace Owners may invite users and [service accounts](/ui/service-accounts/) to work in an organization workspace. In addition to giving the user access to the workspace, the Admin or Owner assigns a [workspace role](/ui/roles/) to the user. The role specifies the scope of permissions for the user within the workspace.
+
+!!! tip "Workspace access in Prefect Cloud organizations"
+    Within an organization, users who are not Admins do not have any default access to organization workspaces. An organization Admin or workspace Owner _must_ invite users to organizations and apply workspace roles to users.
+
+In an organization workspace, select **Workspace Sharing** to manage users and service accounts for the workspace. If you've previously invited users and service accounts, you'll see them listed.
+
+![Managing sharing in a workspace in the Prefect Cloud UI.](/img/ui/workspace-sharing.png)
+
+To invite a user to become a workspace collaborator, select the Members **+** icon. You can select from a list of existing organization members. You can also enter the email address of the person you'd like to invite. If the user does not already have a Prefect Cloud account, they will be able to create one when accepting the invitation.
```

Comment:
In personal accounts, you can add an arbitrary email, in orgs, you cant quite do that yet, but I will file a backlog ticket to enable. You have to add users to your org, and then your workspace.